test/WebJobs.Script.Abstractions/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+    "parallelizeTestCollections": false
```

Comment:
Any reason we don't want to parallelize these?

Reply:
Ah good catch, copy/paste error - no reason not to parallelize